test/common/upstream/health_check_fuzz_test.cc

```diff
@@ -15,9 +15,28 @@ DEFINE_PROTO_FUZZER(const test::common::upstream::HealthCheckTestCase input) {
     return;
   }
 
-  HealthCheckFuzz health_check_fuzz;
-
-  health_check_fuzz.initializeAndReplay(input);
+  switch (input.health_check_config().health_checker_case()) {
```

Comment:
Hey, you're totally right! I tried to do this for a while, but I kept running into problems because HealthCheckFuzz is a virtual class. I remember you telling me in standup that for lb, other modules view this as a black box with the same interface, which is implemented in a specific way. Let me take a look around the codebase for examples of that (which would be instantiating a virtual class)